.travis.yml

```
@@ -12,8 +12,7 @@ addons:
 
 env:
   global:
-  - DJANGO_SETTINGS_MODULE=nrc.conf.jenkins
-  - SECRET_KEY=dummy
+  - DJANGO_SETTINGS_MODULE=nrc.conf.envs.ci
```

```suggestion
  - DJANGO_SETTINGS_MODULE=nrc.conf.ci
```
After talking things through with Joeri, we decided to keep the actual environments in the top-level and the supporting files in the `includes` package